server/sparql-queries.js

```diff
@@ -21,6 +22,7 @@ module.exports = {
             FILTER(LANG(?gloss) = "${languageCode}")
             MINUS { ?senseId wdt:${itemForThisSensePid} [] }
             MINUS { ?senseId wdt:${translationPid} [] }
+            FILTER(?senseId NOT IN (${prefixedSensIdsToSkip.join(', ')}))
```

Comment:
The query will look slightly weird if `senseIdsToSkip` is empty, but it looks like `FILTER(?x NOT IN ())` is valid SPARQL (or at least WDQS accepts it).